CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## 0.4.1 (UNRELEASED)
 
+__BACKWARDS INCOMPATIBILITIES:__
```

Comment:
I would make it default as true and let people who want to disable it disable it.